blocks/columns/columns.js

```diff
@@ -13,6 +13,12 @@ export default function decorate(block) {
           picWrapper.classList.add('columns-img-col');
         }
       }
+      if (block.classList.contains('clickable-images')) {
```

Comment:
This should be handled in css by making the anchor tag width 100% height 100% and position absolute and displayed over the image. There shouldn't be an event listener for the click event.